eclipse-collections-code-generator/src/main/resources/test/bag/mutable/primitiveHashBagTest.stg

```diff
@@ -82,6 +85,18 @@ public class <name>HashBagTest extends AbstractMutable<name>BagTestCase
         <wrapperName> sum = hashBag.injectInto(<wrapperName>.valueOf(<(literal.(type))("4")>), (<wrapperName> result, <type> value) -> <wrapperName>.valueOf((<type>) (result + value)));
         Assert.assertEquals(<wrapperName>.valueOf(<(literal.(type))("17")>), sum);
     }
+
+    @Override
+    @Test
+    public void selectUnique()
```

Comment:
this override can probably be dropped since functionality is already tested by `AbstractMutable<name>BagTestCase`. Or may be I am missing something, please correct me if I am wrong.

Reply:
I agree, the one in `AbstractMutable<name>BagTestCase` already covers the tested functionality.